scipy/_lib/_testutils.py

```diff
@@ -72,7 +72,16 @@ def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
         pytest_args = ['--showlocals', '--tb=short']
 
         if doctests:
-            raise ValueError("Doctests not supported")
+            pytest_args += [
+                "--doctest-modules",
+                "--ignore=scipy/interpolate/_interpnd_info.py",
+                "--ignore=scipy/_lib/array_api_compat",
+                "--ignore=scipy/_lib/highs",
+                "--ignore=scipy/_lib/unuran",
+                "--ignore=scipy/_lib/_gcutils.py",
+                "--ignore=scipy/_lib/doccer.py",
+                "--ignore=scipy/_lib/_uarray",
```

Comment:
can we get rid of the errors from the files which aren't submodules / vendored?

Reply:
Probably. Together with https://github.com/ev-br/scpdt/issues/94
I'd defer it to a follow-up though.

Reply:
That said, if you feel like tackling it in this PR Lucas, I won't mind at all :-).

Reply:
don't tempt me... (no promises but maybe)